subscription-manager.spec

```diff
@@ -178,6 +191,17 @@ Requires: librsvg2
 %description -n subscription-manager-firstboot
```

Comment:
I think the current version will build initial-setup instead of firstboot on RHEL7 or newish fedora.
If we rebase 7.0/7.1 builds and want to build firstboot there, spec file will need some more tweaks.